packages/services/f-feature-management/src/createFeatureManagementInstance.js

```diff
@@ -0,0 +1,25 @@
+import { init as initFeatures } from './featureGetter';
+import { setContextGetter } from './contextGetter';
+import { setLogger } from './logger';
+import FeatureManager from './FeatureManager';
+
+const instance = null;
+
+/**
+ * Returns singleton FeatureManagement object + initialises components on first use.
+ * @param {object} settings
+ * @returns Singleton FeatureManagement object
+ */
+export default function (settings) {
+    if (instance) return instance;
+
+    if (settings.logger) {
+        setLogger(settings.logger);
+    }
+
+    initFeatures(settings.json);
```

Comment:
What produces the JSON here - I can see it gets passed in, does the client application create it?

Reply:
We're basically sidestepping that decision for this initial PR.  But yes if it were to be used as it currently is, the application would pass it in.  For the potential SearchWeb prototype integration this would actually work fine.

Reply:
Okay cool makes sense; yeah doesn't seem wise to stay that way beyond prototype